src/components/Layout/Navbar/index.tsx

```diff
@@ -1,9 +1,54 @@
 import React from "react";
+import Link from "next/link";
+import classNames from "classnames";
+
+import ConunLogo from "@/assets/icons/conun-logo-head-bar.svg";
 
 import styles from "./Navbar.module.scss";
 
 function Navbar() {
-  return <div className={styles.Navbar}>NAV BAR</div>;
+  const PAGES = [
```

Comment:
What if we add this const to a const file instead?

Reply:
Yes! It is done!